src/postgres/contrib/yb_auh/yb_auh.c

```diff
@@ -212,7 +212,8 @@ static void tserver_collect_samples(TimestampTz auh_sample_time, uint16 num_rpcs
     if(random() <= sample_rate * MAX_RANDOM_VALUE){
       auh_entry_store(auh_sample_time, rpcs[i].metadata.top_level_request_id,
                     rpcs[i].metadata.current_request_id, rpcs[i].wait_status_code,
-                    rpcs[i].aux_info.tablet_id, rpcs[i].metadata.top_level_node_id,
+                    (rpcs[i].aux_info.tablet_id[0] == '\0' ? rpcs[i].aux_info.table_id : rpcs[i].aux_info.tablet_id),
```

Comment:
So we seem to be populating the Aux with the table id, if the tablet-id is null.

Sounds like this should be the case -- tablet id being full while in a CQL state ... but I don't think this should be something we rely on. 

What if a thread starts processing a cql-request -- enters the local-tserver mode, and then gets back to the cql mode. Can it have both tablet-id and table-id in the aux_info?

A cleaner solution will be to have a few boolean functions

ShouldExportTabletId(..) /ShouldExportTableId(WaitEvent) ... type function to export one or the other depending on what wait-state we are in.